documentation/docs/15-configuration.md

```diff
@@ -268,6 +269,8 @@ See [Prerendering](/docs/page-options#prerender). An object containing zero or m
     export default config;
     ```
 
+- `origin` — the value of `url.origin` during prerendering
```

Comment:
How about something like?
```suggestion
- `origin` — the value of `url.origin` to use during prerendering. E.g. it can be used if the origin is embedded in the page contents
```

Reply:
```suggestion
- `origin` — the value of `url.origin` during prerendering; useful if it is included in rendered content
```